pwndbg/commands/__init__.py

```diff
@@ -112,7 +112,10 @@ def __init__(
         if command_name is None:
             command_name = function.__name__
 
-        super().__init__(command_name, gdb.COMMAND_USER, gdb.COMPLETE_EXPRESSION, prefix=prefix)
```

Comment:
I guess `prefix` is unused now? Do we want it to be used or should we remove it? I did a quick grep and did not see anyone using it in the codebase, but now that I know it exists it seems useful.

(I'm also surprised the linter didn't flag this unused variable...)

Reply:
Yeah, I did the same check when I was making that change, and it seems that it was never really used. The only place `Command` was ever instanced was through `ArgparsedCommand`, and it just has it be always `False`. I didn't remove it mostly because I failed to realize that call was the only place where it got used.

And, personally, while I think it might seem pretty useful at a first glance, every command in Pwndbg where that could get used - like in `track-got`, for instance - already uses `argparse.ArgumentParser` to achieve the same effect, so I'd say its probably for the best that we avoid it, since using `ArgumentParser` for that frees us from having to deal with subcommands in the debugger-agnostic API.